README.md

```diff
@@ -31,10 +31,14 @@ make services-invoke-publish
 ## Dev setup
 
 ```sh
+# Make sure Ruby is installed
```

Comment:
When you run make dev-static it invokes `ruby -run -ehttpd ./dist/static-site/` to server the site. So it's needed for the dev setup.